.codeclimate.yml

```diff
@@ -1,6 +1,9 @@
 plugins:
   codenarc:
     enabled: true
+    checks:
```

Comment:
out of scope of PR

Reply:
full ACK --> there is a dedicated PR for that #375 .
This can be seen in this PR since otherwise this PR here get a negative vote from codeclimate.
We should agree (of possible) on the other PR first and merge before this here.